mysql-test/suite/tianmu/t/bit_type.test

```diff
@@ -0,0 +1,95 @@
+--source include/have_tianmu.inc
+
```

Comment:
Please use your own database instead of test database to avoid conflict when paralle running test cases；

Reply:
Done.